charts/catalog/templates/apiregistration-and-cert-secret.yaml

```diff
@@ -1,3 +1,8 @@
+{{- $ca := genCA "svc-cat-ca" 3650 }}
```

Comment:
I would be happy to keep this file just named `apiregistration.yaml` for brevity.  Your choice, you did all this work, you should get to pick the name of the file :)  Just my $0.02 I definitely agree that these belong in the same file.